recipes/yaq-traits/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "yaq-traits" %}
+{% set version = "2020.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d66163d8c656a7f1bd9cfd5bb8bedbebe2b62f50c28701b73e54a3019df2723f 
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - yaq-traits=yaq_traits.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - click
+    - colorama
+    - fastavro
+    - prettytable
+    - toml
+    - flit
+    - pip
+    - python >=3.6
+  run:
+    - click
+    - colorama
+    - fastavro
+    - prettytable
+    - python >=3.6
+    - toml
```

Comment:
Is toml needed at runtime?

Reply:
yes? we use it within the package: https://gitlab.com/yaq/yaq-traits/-/blob/master/yaq_traits/_compose.py#L6